Attribution.txt

```diff
@@ -1,72 +1,46 @@
-This software includes code copyrighted from libcoap as follows:
+The following open source projects are referenced by CoAP Device Service:
 
-Copyright (c) 2010--2017, Olaf Bergmann and others
-All rights reserved.
+glibc (GNU LGPL v2.1) https://www.gnu.org/software/libc/
+https://sourceware.org/git/?p=glibc.git;a=blob_plain;f=COPYING.LIB;hb=refs/heads/master
 
-Redistribution and use in source and binary forms, with or without
-modification, are permitted provided that the following conditions are
-met:
+libmicrohttpd (GNU LGPL v2.1) https://www.gnu.org/software/libmicrohttpd/
```

Comment:
@JamesKButcher , @jpwhitemn , Is `GNU LGPL v2.1` compatible with our Apache 2.0 licence?